Configs/.config/waybar/modules/wallchange.jsonc

```diff
@@ -1,5 +1,5 @@
     "custom/wallchange": {
-        "format": "{}",
+        "format": "{}",
```

Comment:
this icons is for change **theme** not for the wallpaper changer

Reply:
Your right - sorry been a long day.